src/calculateRentalCost.js

```diff
@@ -4,7 +4,29 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_COST = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let totalAmount = 0;
```

Comment:
but why? u can return `defaultPrice - discount` immediately. U don't need a variable for this